xml/convert/oscal_catalog_json-to-xml-converter.xsl

```diff
@@ -1289,6 +1289,7 @@
          <xsl:apply-templates mode="#current" select="*[@key=('title')]"/>
          <xsl:apply-templates mode="#current" select="*[@key=('param', 'parameters')]"/>
          <xsl:apply-templates mode="#current" select="*[@key=('prop', 'properties')]"/>
+         <xsl:apply-templates mode="#current" select="*[@key=('annotation', 'annotations')]"/>
          <xsl:apply-templates mode="#current" select="*[@key=('link', 'links')]"/>
          <xsl:apply-templates mode="#current" select="*[@key=('part', 'parts')]"/>
          <xsl:apply-templates mode="#current" select="*[@key=('control', 'controls')]"/>
```

Comment:
@brianrufgsa You shouldn't commit any changes against files in the xml, json, or content directories, since these files are generated by the CI/CD. Can you revert these files?